hooks/post_gen_project.py

```diff
@@ -101,6 +101,7 @@ def init_git(message):
 
     # docs folder
     remove_files("docs")
+    remove_files("lib/api/docs")
```

Comment:
Should we remove the `lib/api` entirely? I tried generating the `Web only` variant and the `api` folder is still there 💭 

```suggestion
    remove_files("lib/api")
```

![Screenshot 2566-08-25 at 17 56 46](https://github.com/nimblehq/gin-templates/assets/1772999/8170d753-4111-4da0-9256-9c7fdbf67b35)
![Screenshot 2566-08-25 at 17 57 04](https://github.com/nimblehq/gin-templates/assets/1772999/7fd76121-d7eb-474b-b6f8-73394b97bf2c)

Reply:
Fixed in https://github.com/nimblehq/gin-templates/pull/107/commits/78e400bc8a3087fb7b483ad7c2a860c1b896e4a6